README.md

```diff
@@ -246,6 +246,7 @@ Some nice to have but probably separate items:
 ### Building the Runtime
 
 Contributors and other advanced users may want to build the runtime from source. To do so, you'll need to have Maven installed.
+`Java version 11` required for a proper build. You can download and install [Java 11 Temurin](https://adoptium.net/temurin/releases/?version=11)
```

Comment:
Have you faced issues with other Java versions?
The requirement should be 11+ I'm currently on 17 and I use 21 sometimes too.